modules/@apostrophecms/notification/ui/apos/apps/ApostropheNotification.js

```diff
@@ -0,0 +1,56 @@
+import Vue from 'apostrophe/vue';
+
+export default function() {
+  /* eslint-disable no-new */
+  return new Vue({
+    el: '#apos-notification',
+    data () {
+      return {
+        notifications: []
+      };
+    },
+    computed: {
+      apos() {
+        return window.apos;
+      }
+    },
+    async mounted() {
+      if (apos.scene !== 'apos') {
+        return;
+      }
+
+      this.notifications = await this.poll();
+      setInterval(async() => {
+        this.notifications = await this.poll();
+      }, 10000);
```

Comment:
I believe the A2 interval is every 5 seconds.

I see that you aren't dealing with the activity log window yet, and that should probably get opened as a second ticket, but be aware that they will need to share the polling response via the bus in order to avoid the performance hit of double polling so often.

Reply:
You should add a destroy handler to clean up the interval.

Reply:
Done